utils/wasm-instrument/src/syscalls.rs

```diff
@@ -293,15 +295,13 @@ impl SysCallName {
             Self::MessageId => SysCallSignature::gr([Ptr(PtrInfo::new_mutable(PtrType::Hash(
                 HashType::MessageId,
             )))]),
+            Self::ExecSettings => {
+                SysCallSignature::gr([Version, Ptr(PtrInfo::new_mutable(PtrType::Buffer))])
+            }
             Self::Read => SysCallSignature::gr([
                 MessagePosition,
                 Size,
-                // TODO #3375, the PtrType::BlockNumber is incorrect here.
```

Comment:
where did todo disappeared?

Reply:
as we discussed, I still think that it should be buffer start
if wasmgen buffer start requires len that it should be edited into BufferStart and SizedBufferStart

Reply:
> as we discussed, I still think that it should be buffer start
if wasmgen buffer start requires len that it should be edited into BufferStart and SizedBufferStart

just make some renaming there